flank-scripts/src/test/kotlin/flank/scripts/exceptions/FlankScriptsExceptionMappersTest.kt

```diff
@@ -0,0 +1,115 @@
+package flank.scripts.exceptions
+
+import com.github.kittinunf.fuel.core.FuelError
+import com.github.kittinunf.result.Result
+import com.github.kittinunf.result.failure
+import com.google.common.truth.Truth.assertThat
+import io.mockk.every
+import io.mockk.mockk
+import org.junit.Test
+
+class FlankScriptsExceptionMappersTest {
+
+    @Test
+    fun `Should properly map client error`() {
+        // given
+        val fuelMockedError = mockk<FuelError> {
+            every { response.statusCode } returns 404
+        }
+        val result: Result<Any, FuelError> = Result.error(fuelMockedError)
+        val expectedException = Exception("test")
+
+        // when
+        val (_, actualException) = result.mapClientError { expectedException }
+
+        // then
+
```

Comment:
extra newline?

Reply:
fixed 👍  and also added `detekt` to project with the same configuration as flank